.github/workflows/sync-tokens-to-figma.yml

```diff
@@ -15,18 +15,20 @@ jobs:
         with:
           node-version: '20.19.x'
 
-      - name: Set NPM version
-        run: npm install -g npm@9.5.1
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: 10
 
       - name: Clone repo
         uses: actions/checkout@v3
 
       - name: Install dependencies
-        run: yarn install
+        run: pnpm install
         working-directory: design-tokens
 
       - name: Sync tokens to Figma file
-        run: yarn sync-tokens-to-figma
+        run: pnpm sync-tokens-to-figma
         working-directory: design-tokens
```

Comment:
Should we be using filter instead of working directory?

Reply:
Optional. We can remove the `working-directory` line and use `--filter` or use this style. Depends what folks prefer.